data/test_config/config_defaults.yml

```diff
@@ -21,3 +21,12 @@ pipeline: []
 #     ambiguity_threshold: 0.1
 
 data:
+policies:
```

Comment:
can we remove this?

Reply:
This is automatically added when you run the tests.

Reply:
So either everyone who runs tests locally has to always roll back this file, or we just commit it once.

Reply:
I think the test which causes this should be changed then. I'd not commit this.

Reply:
We don't have a test for this. I just discovered it when I was implementing something new. I'll undo the deletion anyway.

Reply:
Oh! I thought this comment was about the `__hash__` thing. Ok, I have to think about this.

Reply:
The problem is that this comes just from `RasaFileImporter` when used with the `config_file` argument. I might have to change that component.

Reply:
Ah! `training_type=TrainingType.NLU` solves the issue :)